editor/src/messages/portfolio/document/graph_operation/graph_operation_message_handler.rs

```diff
@@ -103,10 +103,73 @@ impl MessageHandler<GraphOperationMessage, GraphOperationMessageData<'_>> for Gr
 					modify_inputs.brush_modify(strokes);
 				}
 			}
+			GraphOperationMessage::MoveUpstreamSiblingsToChild { new_parent, mut upstream_sibling_ids } => {
```

Comment:
Should the message signature have this be `mut` because the intention is for this to mutate its caller's data?